ci/run_integration_test.sh

```diff
@@ -15,20 +17,26 @@ pip install awscli
 pip install https://github.com/scylladb/scylla-ccm/archive/master.zip
 
 # download version
-LATEST_MASTER_JOB_ID=`aws --no-sign-request s3 ls downloads.scylladb.com/relocatable/unstable/master/ | grep '2020-' | tr -s ' ' | cut -d ' ' -f 3 | tr -d '\/'  | sort -g | tail -n 1`
-AWS_BASE=s3://downloads.scylladb.com/relocatable/unstable/master/${LATEST_MASTER_JOB_ID}
+LATEST_MASTER_JOB_ID=`aws --no-sign-request s3 ls downloads.scylladb.com/relocatable/unstable/${BRANCH}/ | grep '2020-' | tr -s ' ' | cut -d ' ' -f 3 | tr -d '\/'  | sort -g | tail -n 1`
+AWS_BASE=s3://downloads.scylladb.com/relocatable/unstable/${BRANCH}/${LATEST_MASTER_JOB_ID}
 
 aws s3 --no-sign-request cp ${AWS_BASE}/scylla-package.tar.gz .
 aws s3 --no-sign-request cp ${AWS_BASE}/scylla-tools-package.tar.gz .
 aws s3 --no-sign-request cp ${AWS_BASE}/scylla-jmx-package.tar.gz .
 
-ccm create scylla-driver-temp -n 1 --scylla --version unstable/master:$LATEST_MASTER_JOB_ID \
+ccm create scylla-driver-temp -n 1 --scylla --version unstable/${BRANCH}:$LATEST_MASTER_JOB_ID \
  --scylla-core-package-uri=./scylla-package.tar.gz \
  --scylla-tools-java-package-uri=./scylla-tools-package.tar.gz \
  --scylla-jmx-package-uri=./scylla-jmx-package.tar.gz
 
 ccm remove
 
 # run test
-export SCYLLA_VERSION=unstable/master:$LATEST_MASTER_JOB_ID
-PROTOCOL_VERSION=4 EVENT_LOOP_MANAGER=asyncio pytest --import-mode append tests/integration/standard/
+
+echo "export SCYLLA_VERSION=unstable/${BRANCH}:${LATEST_MASTER_JOB_ID}"
+echo "PROTOCOL_VERSION=4 EVENT_LOOP_MANAGER=asyncio pytest --import-mode append tests/integration/standard/"
+export SCYLLA_VERSION=unstable/${BRANCH}:${LATEST_MASTER_JOB_ID}
+export MAPPED_SCYLLA_VERSION=4.1.0
+PROTOCOL_VERSION=4 EVENT_LOOP_MANAGER=asyncio pytest -rf --import-mode append tests/integration/standard/test_authentication.py tests/integration/standard/test_cluster.py tests/integration/standard/test_concurrent.py
```

Comment:
it would be nice if we run the shared aware test ones...

how long it take for the whole standard suite to run ?

Reply:
Also an idea, we could splint into few stages in travis, each stage is limited to ~50min

Reply:
46 minutes is totall time in python-driver-matrix tests